core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -394,6 +401,7 @@ class ContainerPool(
     private val warmNodejsKey = WarmNodeJsActionContainerId
     private val nodejsExec = NodeJS6Exec("", None)
     private val WARM_NODEJS_CONTAINERS = 2
+    private val RM_THRESHOLD = 4
```

Comment:
Jeremias is right, just an arbitrary threshold as an additional safeguard to prioritize the rm ahead of stem cell container.  The accounting with toBeRemoved already protects against the main bug.